docs/configuring-playbook-synapse-s3-storage-provider.md

```diff
@@ -43,10 +43,18 @@ matrix_synapse_ext_synapse_s3_storage_provider_config_storage_class: STANDARD #
 # This only works on AWS when your server is hosted on an EC2 instance with the correct instance profile set.
 # Uncomment the variable below to use it.
 # matrix_synapse_ext_synapse_s3_storage_provider_config_ec2_instance_profile: true
-
-# For additional advanced settings, take a look at `roles/custom/matrix-synapse/defaults/main.yml`
 ```
 
+### Extending the configuration
+
+There are some additional things you may wish to configure about the server.
+
+Take a look at:
+
+- `roles/custom/matrix-synapse/defaults/main.yml` for some variables that you can customize via your `vars.yml` file
```

Comment:
I'll merge this as-is, but I suppose similar `roles/` references can be turned into links to make it easier for users to jump to the files in question.

That said, I hope relative links bode well with translations (due to translations living in another directory and it being related to `roles/` in a different way)